cloudflare/src/framework/blocking_api.rs

```diff
@@ -0,0 +1,97 @@
+use crate::framework::{Environment, auth, endpoint, response};
```

Comment:
Self-comment: new file, albeit all the comments were just moved, none of the code is new.